src/mtcp/mtcp_split_process.c

```diff
@@ -1,3 +1,4 @@
+// FIXME:  Why do we have all these libc headers if we're not using libc here?
```

Comment:
You are right, we don't need all the libc headers here. We only need a few of them in order to compile this file.

Here's a minimal list of header includes we do need in order to compile this file; every other include can be removed.

```C
#include <asm/prctl.h> // For ARCH_GET_FS, ARCH_SET_FS
#include <sys/prctl.h> // For ARCH_GET_FS, ARCH_SET_FS
#include <sys/personality.h> // For ADDR_NO_RANDOMIZE
#include <ucontext.h> // For ucontext_t
#include <sys/uio.h> // For struct iovec
#include <link.h> // For ElfW, auxv_t
```